### ISSUE TITLE
Crash: using Network code, Bomberman demo, Windows only

### ISSUE THREAD

**Issue by artiescie** [godotengine/godot#40222]
*RELATED EXISTING ISSUES:*
These report issue on Win10, but don't narrow the problem to Windows.  
Also, the current report simplifies how to reproduce.
--> godot client crash when networking is introduced and hit play #37544
   Apr 2, 2020
--> Multiplayer Bomber example crashes #38549
May 7, 2020

**OS/DEVICE**
Windows only, tested 2 windows machines: Win8.1 and Win10.  
No such issue found on Android export, where the current version and prior versions were tested.
No other OS tested.

**GODOT VERSION:** 3.2.2 stable

**DESCRIPTION:**
CPU goes quickly to around 20%.  Eventually, machine crashes, all without doing anything in the user interface.

**TO REPRODUCE:**
Load the multiplayer bomberman demo.  Run from the IDE.  Watch the first "lobby" screen display.  Do nothing else other than monitor CPU usage.

CPU begins to rise in a few seconds.  On my machine (Lenovo T530 Intel i7-3740QM CPU$2.70 GHz, 16.0GB RAM, win 8.1) it will go to 18-20%.  If left for 15 minutes or so, machine will usually crash.

Same if you Export to Windows, and run the resulting EXE.  

If you run from IDE, you have the debug "Pause Scene (F7)" button, that should stop any code running at project level.  If you push this button, it has no effect.  Evidently, it's executing library code out of scope for the debugger.

**MIN REPRODUCTION PROJECT:**
Not sure if it's minimal ;-) but everyone has access to it: the distributed Multiplayer Bomberman demo project.

No other demo project in the distributed demo package will cause this CPU issue, I loaded them all.

Android runs the same project (exported for Android) with no problem.  Tested Android version 9 and several prior versions.

Also #37544 (referenced above) finds that just adding networking caused crash.

Put together, leads to suspicion: that the networking libraries may be buggy for Windows platform only.

Same issue was found in version 3.2.1 (stable) (which was current when I started using Godot just recently)

**Comment by Faless** [comment]
Most likely fixed in recent Godot 3 and 4 after #38119 . Closing as there have been no recent reports of such crashes, but feel free to comment (or open a new issue) if you are still experiencing it.
